kf_lib_data_ingest/target_apis/kids_first_original_dataservice.py

```diff
@@ -0,0 +1,768 @@
+"""
```

Comment:
Tried ingest with cat study (`kidsfirst ingest tests/data/SD_ME0WME0W`) and I'm getting a 400
```
{
  "_status": {
    "code": 400,
    "message": "could not create phenotype: {'age_at_event_days': ['Not a valid integer.']}"
  }
}
```
Idk if you already did this but maybe run cat study and one complex study to shake out any bugs in the mapper classes.

Reply:
huh

Reply:
TBH I forgot about the swagger check from before, but I think we should talk about the safety of the old behavior. It looks like that study returns float values and then the old swagger fetch code would automatically coerce the values to ints based on the swagger type field, but should we really be coercing values like that?

Reply:
Like, I _think_ it's safe to blindly cast without looking on the premise that only compatible types will cast without erroring, but I would like confirmation.

Reply:
Hmm I think we could blindly cast, but what if we left this up to the developer of the target API config?

They should know the types of each attribute in the target service so couldn't we just add `some_coerce_func(row.get(CONCEPT.PHENOTYPE.AGE_EVENT_DAYS))` in the kids first target api config? I think you already have a function for this somewhere

Reply:
> what if we left this up to the developer of the target API config?

I'm assuming that the swagger auto-coercion would happen in the dataservice's target api module and not in load.py, so in a way it would be.

Reply:
Ohhh I see, ok 👍